nixos/modules/services/x11/xserver.nix

```diff
@@ -725,7 +725,7 @@ in
     systemd.defaultUnit = mkIf cfg.autorun "graphical.target";
 
     systemd.services.display-manager =
-      { description = "X11 Server";
+      { description = "display manager";
```

Comment:
I think these descriptions are typically capitalized. See `systemctl` output for examples. (I think I read so too in systemd documentation a while back, but I'm too lazy to locate the reference.)